iOS-Weather-App/Controllers/WeatherViewController/WeatherViewController.swift

```diff
@@ -53,7 +39,14 @@ final class WeatherViewController: UIViewController{
     }
 
     override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
-        setupBackground()
+        if previousTraitCollection?.userInterfaceStyle == .dark {
+            backgroundImage.image = UIImage(named: "dayBackground")
+        }
+        else {
+            backgroundImage.image = UIImage(named: "nightBackground")
+        }
+        backgroundImage.contentMode = UIView.ContentMode.scaleAspectFill
+        view.insertSubview(backgroundImage, at: 0)
```

Comment:
you can setup this line at once from viewDidLoad and change an image when it need it